dot/peerset/peerstate.go

```diff
@@ -261,12 +263,15 @@ func (ps *PeersState) addNoSlotNode(idx int, peerID peer.ID) {
 
 func (ps *PeersState) removeNoSlotNode(idx int, peerID peer.ID) {
 	if _, ok := ps.sets[idx].noSlotNodes[peerID]; !ok {
+		logger.Debugf("peer %s already does not exist in no slot node", peerID)
 		return
 	}
 
 	delete(ps.sets[idx].noSlotNodes, peerID)
 	n, err := ps.getNode(peerID)
 	if err != nil {
+		// TODO: Return the error
```

Comment:
do you want to open an issue for this or do follow up PR to fix this?

Reply:
done